scrapyd_k8s/api.py

```diff
@@ -148,5 +146,7 @@ def run():
     if config_username is not None and config_password is not None:
         enable_authentication(app, config_username, config_password)
 
+    launcher.enable_joblogs(config)
```

Comment:
What about putting this in a conditional like  `if config.joblogs() is not None`, I think that makes it more readable, now I think each time I read it: hey, is joblogs always enabled or not?
This would also avoid a warning about joblogs not being enabled when not configured. I think of this feature as an add-on, not as something that everyone will use.